### ISSUE TITLE
How do we change the path s3 

### ISSUE THREAD

**Issue by surajsonee** [joel-simon/ganbreeder#15]
I have already change the path of s3 and bucket name, but when i run the node make_random.js its still getting files from some other place.

**Comment by kartikeyporwal** [comment]
@surajsonee Hi, did you get this working? I also need to get this change.

**Comment by surajsonee** [comment]
> @surajsonee Hi, did you get this working? I also need to get this change.

yes its working, Now I am looking to train the model with my own data-sets.

**Comment by AkoZ** [comment]
@surajsonee 
> 
> yes its working, Now I am looking to train the model with my own data-sets.

hey, please, may you give an ez step by step manner on how to get the image then the training ?
for now, as docker is ok.. i can't get the localhost frontend or backend start, it just end with an "all done" but i can't find the calculated image as localhost is refused.

**Comment by surajsonee** [comment]
> @surajsonee
> 
> > yes its working, Now I am looking to train the model with my own data-sets.
> 
> hey, please, may you give an ez step by step manner on how to get the image then the training ?
> for now, as docker is ok.. i can't get the localhost frontend or backend start, it just end with an "all done" but i can't find the calculated image as localhost is refused.

please contact me in skype: suraj.prasad1990

**Comment by AkoZ** [comment]
@surajsonee oh , i'vent seen your message. Shame. As for now i would have to get back for all and re-learn the process... abandonned.
but to precise, as i re-read the "localhost" problem is recurrent for the GAN extract: i've the same stupid pb for an other algo !!
* so i'll concentrate on this -localhost-  with some firewall blockage, i'll find that.. 
then i'll come bck as it seems not to be ganbreeder pb !
At all thank you very much for your reply !
